test/Test/Fission/DNS.hs

```diff
@@ -0,0 +1,46 @@
+module Test.Fission.DNS (tests) where
+
+import           Test.Fission.Prelude
+
+import qualified Fission.DNS          as DNS
+import qualified RIO.NonEmpty.Partial as NonEmpty.Partial
+
+tests :: IO TestTree
+tests =
+  testSpec "Fission.DNS" do
+    describe "splitRecord" do
+      it "does not split a string under 256 chars" do
+        DNS.splitRecord "asdf" `shouldBe` pure "asdf"
```

Comment:
Why not make these prop tests? You can control the size a couple ways, but the most straightforward without having to learn the library would be something like:

```haskell
-- Helper type, maybe right in the test file
newtype SmallText = SmallText Text
  deriving newtype (Show, Eq)

instance Arbitrary SmallText where
  arbitrary = do
    txt <- arbitrary
    return . SmallText $ Text.take 256 txt

-- Tests

testSpec “Fission.DNS” d
  describe “...” do
    context “under 256 characters” do
      itsProp’ “length does not change ” \SmallText txt ->
        DNS.splitRecord txt `shouldBe pure txt
````

Reply:
cool :+1:  this is really helpful

Should I be prop testing longer strings as well? seems like i'll essentially be re-coding the function to ensure that it works, but maybe that's good?